### PR TITLE
554: Fixes new admin project icons

### LINKED ISSUE #554: Bug: Activating and deactivating projects in new_admin not working
When you edit a project, and try to activate it, or deactivate it, it changes correctly in the database, but the icon in the view used to show if a project is active is not changing correctly: 

How to replicate this bug: 

Go to new_admin/projects, pick a project to edit. 

Try deactivating/activating it, then save the changes. 

Go to the index page, or the show page of that project, check if the icon changed (it didn't).

Check in the rails console to see if the active attribute changed (it did).

### PR DESCRIPTION
**Pull request by ghost**
Resolves #554

image

### CODE REVIEW

File: app/views/new_admin/projects/_project_details.html.erb [hunk 1]
```diff
@@ -62,7 +62,7 @@
         <td class="px-4 py-3">
           <div class="flex items-center text-sm">
             <p class="text-sm text-gray-600 dark:text-gray-400 font-semibold">
-              <%= inline_svg_tag 'check_circle_fill.svg', class: "h-6 w-6 #{project.active_class}" %>
+              <%= inline_svg_tag "#{project.active ? 'check' : 'x' }_circle_fill.svg", class: "h-6 w-6" %>
```
Comment (nehamamandelbaum) [inline app/views/new_admin/projects/_project_details.html.erb:65]: Sugestão: Nas novas PRs estamos usando o heroicons https://github.com/bharget/heroicon ao invés do SVG em si.

**Review by nehamand — APPROVED** [review]

**Comment by nehamamandelbaum** [comment]
Alguns testes do project decorator estão quebrando, seria bom ver isso https://github.com/Codeminer42/Punchclock/actions/runs/7119000719/job/19383164783?pr=643

**Review by nehamand — CHANGES_REQUESTED** [review]
Alguns testes do project decorator estão quebrando, seria bom ver isso https://github.com/Codeminer42/Punchclock/actions/runs/7119000719/job/19383164783?pr=643

File: app/assets/images/x_circle_fill.svg
Comment (adbatista) [inline app/assets/images/x_circle_fill.svg:1]: this should not be necessary, the gem should provide this

**Review by nehamand — APPROVED** [review]

**Pull request merged**